markov-chain/main.py

```diff
@@ -30,10 +29,9 @@ def generate_paragraph(m, n_sentences, max_words_per_sentence):
     text = ""
     for i in range(n_sentences):
         sentence = m.generate_sentence(max_words_per_sentence)
-        text = text + " " + sentence
+        text += " " + sentence
```

Comment:
I don't feel particularly strong about either `text = text + ...` or `text += ...`, but whichever you choose, be consistent.